### PR TITLE
fix(Table): Added localization for selected rows count

### LINKED ISSUE #936: Table: row selection count needs to be localized
_Originally posted in https://github.com/iTwin/iTwinUI-react/pull/837#discussion_r1012873826_

### PR DESCRIPTION
**Pull request by elephantcatdog**
Fixed some things I found while writing the migration guide.
- Incorrect example for AvatarGroup
- Grammar in comments
- Added localization for selected rows count [(based on this comment)](https://github.com/iTwin/iTwinUI-react/pull/837#:~:text=this%20needs%20to%20be%20localized)

image

## Checklist

- [x] Add meaningful unit tests for your component (verify that all lines are covered)
- [x] Verify that all existing tests pass
- [ ] ~Add component features demo in Storybook (different stories)~
- [ ] ~Approve test images for new stories~
- [x] Add screenshots of the key elements of the component

### CODE REVIEW

**Review by gretanausedaite — COMMENTED** [review]
Code looks good. 
Could you add story for localized table? I noticed we don't have example how to do it.

**Comment by gretanausedaite** [comment]
> 

No new images for localized story?

File: apps/storybook/src/Table.stories.tsx [hunk 1]
```diff
@@ -1669,6 +1669,100 @@ Full2.decorators = [
...
+      <Story />
+    </div>
+  ),
+];
+
+Localized.argTypes = {
+  data: { control: { disable: true } },
+};
+Localized.parameters = {
+  docs: { source: { excludeDecorators: true } },
```
[the remark this answers is not in the document]
Comment (elephantcatdog) [reply apps/storybook/src/Table.stories.tsx:1763]: Not sure. I copied the story from the "Paginator" story and changed some things. I assume it's there for a reason, but I couldn't find a lot of info about "excludeDecorators" except in preventing undesirable behavior.

File: apps/storybook/src/Table.stories.tsx [hunk 2]
```diff
@@ -1669,6 +1669,100 @@ Full2.decorators = [
...
+            startIndex: number,
+            endIndex: number,
+            totalRows: number,
+            isLoading: boolean,
+          ) =>
+            isLoading
+              ? `${startIndex}-${endIndex} localized`
+              : `${startIndex}-${endIndex} of localized ${totalRows}`,
+          previousPage: '',
+          nextPage: '',
```
[the remark this answers is not in the document]
Comment (elephantcatdog) [reply apps/storybook/src/Table.stories.tsx:1721]: Whoops, missed that. Updated

File: apps/storybook/src/Table.stories.tsx [hunk 3]
```diff
@@ -1669,6 +1669,100 @@ Full2.decorators = [
...
+        }}
+      />
+    ),
+    [pageSizeList],
+  );
+
+  return (
+    <>
+      <Table
+        emptyTableContent='No data.'
```
[the remark this answers is not in the document]
Comment (elephantcatdog) [reply apps/storybook/src/Table.stories.tsx:1737]: Thank you, updated

**Comment by elephantcatdog** [comment]
> No new images for localized story?

Whoops, fixed

**Review by gretanausedaite — APPROVED** [review]

**Review by r100-stack — APPROVED** [review]

**Pull request merged**